css-three-column-layout/styles.css

```diff
@@ -0,0 +1,71 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  background-color: rgb(25, 150, 218);
+}
+
+#container {
+  margin: auto;
+  width: 900px;
+  display: flex;
+  flex-direction: column;
```

Comment:
This shouldn't be necessary 🤔

Reply:
Should all of this not be necessary? I've taken out lines 11 and 12, to make it have the structure similar to that of the example. But needed to keep lines 13 & 14 to keep it in the desired structure.